fly.toml

```diff
@@ -11,33 +9,8 @@ processes = []
 [env]
   PORT = "8080"
 
-[experimental]
-  allowed_public_ports = []
-  auto_rollback = true
-
-[[services]]
-  http_checks = []
+[http_service]
   internal_port = 8080
-  processes = ["app"]
-  protocol = "tcp"
-  script_checks = []
-
-  [services.concurrency]
-    hard_limit = 25
-    soft_limit = 20
-    type = "connections"
-
-  [[services.ports]]
-    force_https = true
-    handlers = ["http"]
-    port = 80
-
-  [[services.ports]]
-    handlers = ["tls", "http"]
-    port = 443
-
-  [[services.tcp_checks]]
-    grace_period = "1s"
-    interval = "15s"
-    restart_limit = 0
-    timeout = "2s"
```

Comment:
only using necessary configuration based on new template autogenerated for Fly.io in v2 architecture

Reply:
I'm guessing for the same reason, that it's configured already by default, like mentioned here:  https://github.com/upleveled/security-vulnerability-examples-next-js-postgres/pull/85/files#r1190099894